### PR TITLE
Oppa chin choppa

### LINKED ISSUE #707: add blahdns list
https://github.com/ookangzheng/blahdns/tree/master/hosts

### PR DESCRIPTION
**Pull request by bogachenko**
close #707 
@collinbarrett how to come up with a description?

### CODE REVIEW

File: data/FilterList.json [hunk 1]
```diff
@@ -16830,5 +16830,29 @@
     "name": "AdHell Standart",
```
Comment (collinbarrett) [inline data/FilterList.json]: mind fixing the spelling error to "Standard" from your last PR? thanks.

**Comment by collinbarrett** [comment]
As for descriptions, if you can't find anything reasonable on the GitHub repo for the project to copy/paste, I guess just leave blank for now.

File: data/FilterList.json (hunk 1 above)
Comment (bogachenko) [reply data/FilterList.json]: I thought that there are other lists, something like "Full" or something like that, well, okay

File: data/FilterList.json [hunk 2]
```diff
@@ -16827,8 +16827,32 @@
   {
     "id": 1471,
     "homeUrl": "http://getadhell.com",
-    "name": "AdHell Standart",
+    "name": "AdHell list",
```
Comment (collinbarrett) [inline data/FilterList.json]: Oh, you can keep "Standard" in the title. Just requesting to swap the "t" for a "d" so the spelling is correct (matches the `viewUrl` of standar**d**-package.

Comment (bogachenko) [reply data/FilterList.json]: fuck// okey wait

Comment (bogachenko) [reply data/FilterList.json]: I learn English, and spelling is still a complicated thing.

Comment (collinbarrett) [reply data/FilterList.json]: No worries at all. Thanks for all your help.

**Comment by bogachenko** [comment]
uh, such a question, what's with travis? why error?

**Comment by bogachenko** [comment]
what did I do wrong again?

**Comment by bogachenko** [comment]
@collinbarrett reception. why is everything red there?

**Comment by collinbarrett** [comment]
Oh, don't worry about it. I merged it before Travis finished building, so [it looks like](https://travis-ci.org/collinbarrett/FilterLists/builds/502651097) Travis just failed to fetch the PR code.

I think everything is fine.

Thanks.

**Comment by bogachenko** [comment]
okey

**Pull request merged**